README.md

```diff
@@ -5,8 +5,13 @@ Today, The Hack Foundation is the largest fiscal sponsor for teen-led projects i
 
 Every year, tens of thousands of people are impacted by organizations operating under The Hack Foundation. Partnered organizations receive the benefits of 501(c)(3) nonprofit status, administrative and backoffice support, and financial oversight.
 
-# Colophon
-- Tailwind CSS - for styling
-- a raw CSS animation - for that fancy animation at the bottom (check it out [here](/components/AnimationContainer.js))
-
 &copy; 2021 The Hack Foundation. All Rights Reserved.
+
+Yeah! Here's yo' links @sampoder
+https://cloud-mdfqxm18z-hack-club-bot.vercel.app/068747470733a2f2f636c6f75642d387a6c6179706b68692e76657263656c2e6170702f323032302d30372d33305f757734747a623577746a777176756376366d71323266347a36616232346d75662e706e67.png
+https://cloud-mdfqxm18z-hack-club-bot.vercel.app/1ae6618_52a8d7fbf108462a8c540fd13973b879_mv2.jpg
+https://cloud-mdfqxm18z-hack-club-bot.vercel.app/2orpheus_legion.jpg
+https://cloud-mdfqxm18z-hack-club-bot.vercel.app/3dv-hacks.jpg
+https://cloud-mdfqxm18z-hack-club-bot.vercel.app/4teenhacks-li.jpg
+https://cloud-mdfqxm18z-hack-club-bot.vercel.app/50screenshot_2021-04-05_at_7.25.05_pm-6.png
+https://cloud-mdfqxm18z-hack-club-bot.vercel.app/6screenshot_2021-10-03_at_3.29.29_pm.png
```

Comment:
```suggestion
```

Not sure if this was intentionally committed.

Reply:
good catch!